include/swift/Localization/LocalizationFormat.h

```diff
@@ -194,9 +208,10 @@ class YAMLLocalizationProducer final : public LocalizationProducer {
   /// maintained by this producer, callback gets each translation
   /// with its unique identifier.
   void forEachAvailable(
-      llvm::function_ref<void(swift::DiagID, llvm::StringRef)> callback) const;
+      llvm::function_ref<void(swift::DiagID, llvm::StringRef)> callback);
```

Comment:
I had to remove `const` because this method can initialize the  YAMLLocalizationProducer

Reply:
Sounds good!